api/lib/config.js

```diff
@@ -49,6 +49,7 @@ module.exports = (function() {
       mailjetApiKey: process.env.MAILJET_KEY,
       mailjetApiSecret: process.env.MAILJET_SECRET,
       mailjetAccountCreationTemplateId: process.env.MAILJET_ACCOUNT_CREATION_TEMPLATE_ID,
+      mailjetOrganizationInvitationDemandTemplateId: process.env.MAILJET_ORGANIZATION_INVITATION_DEMAND_TEMPLATE_ID,
```

Comment:
problème de naming, en anglais `to demand` c'est `exiger`. Donc là ça me semble un peu fort comme terme 😅`mailjetOrganizationInviteTemplateId` ou `mailjetOrganizationInvitationTemplateId` tout simplement me paraît correct 😄